### DISCUSSION TITLE
pc版给ios版授权通过显示等待ios升级专业版 但是一直无反应

### DISCUSSION THREAD

**Discussion by cyrilpetrilli** [getlantern/lantern-forum-cn#868] [General]
问题如题 还有用邮箱接收激活码会一直显示我们遇到了技术困难 请稍后重试  we're expirencing technical difficulties try again later 
请帮助我解决下  幸苦了 对抗高墙的战士们

**Comment by cyrilpetrilli** [comment]
我自己给捣腾通了，在ios上用邮箱收授权码和专业版赋权的方式都走不通，我用创建帐户密码登录的方式激活了ios专业版，整个过程要保持翻墙状态，其他连接不上的兄弟们可以试下，给一点我自己的建议就是，不要用国内的邮箱，ios搞一个美区账号 ，还有大家不要怪维护组，好多事不是他们不搞定是高墙太可恶，占有的资源不一样能量级别也是不同的。

**Reply by kidokogills** [reply]
怎么创建账户密码登录？

**Reply by cyrilpetrilli** [reply]
ios版本应该有三个授权模式在选项里 有一个帐号密码登录  如果从来没用过就用现在的账号设定一个密码就好

**Comment by MaxMKMax** [comment]
@cyrilpetrilli 哇. 你真棒. 谢谢您帮我们说话. 的确如此, 很多事情不是我们能控制的. 如果您还需要帮忙请您找我. 也谢谢您分享这个方法. 辛苦您了还要自己找解决方案. 谢谢啦!
